### ISSUE TITLE
Does this code support Azure Kinect DK and Hololens2?

### ISSUE THREAD

**Issue by cevo2022** [MarekKowalski/LiveScan3D-Hololens#30]
Does this code support Azure Kinect DK and Hololens2?

**Comment by MarekKowalski** [comment]
Yes, there is a branch for HL2 here and there is a branch for Azure Kinect DK in the LiveScan3D repo. I have not tested them in a while, but they should work.

Thanks,

Marek
